### DISCUSSION TITLE
Asus mobo BIOS upgrade

### DISCUSSION THREAD

**Discussion by manio** [fwupd/fwupd#8125] [Q&A] [answered]
Hi!
Guys, I have an Asus Z170-PRO motherboard with bios from 03/24/2016.
After recent debian kernel upgrade I have a never ending spam of this messages in dmesg:
```
[Mon Nov 18 15:53:18 2024] ACPI BIOS Error (bug): Could not resolve symbol [\_GPE._L69.D1F0], AE_NOT_FOUND (20220331/psargs-332)
[Mon Nov 18 15:53:18 2024] ACPI Error: Aborting method \_GPE._L69 due to previous error (AE_NOT_FOUND) (20220331/psparse-529)
[Mon Nov 18 15:53:18 2024] ACPI Error: AE_NOT_FOUND, while evaluating GPE method [_L69] (20220331/evgpe-511)
```
Thankfully I was able to disable it manually with:
`echo "disable" > /sys/firmware/acpi/interrupts/gpe69`

But I was thinking about upgrading BIOS as I read it might help in this case.

I am totally newbie regarding `fwupd` (I am familiar with "classic" upgrade process with specific manufacturer tools).
Can my motherboard BIOS be upgraded from within linux and fwupd ?
I was reading the documentation about support but there are only two (sic!) Asus devices ?

**Comment by hughsie** [comment, marked as answer]
> but there are only two (sic!) Asus devices

Yes unfortunately the LVFS requires the OEM to upload the firmware image (for legal reasons) and so they can pick and choose what models are supported in Linux.
